include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -259,15 +272,22 @@ class inplace_vector : public inplace_vector_base<T, Capacity> {
       base::uninitialized_copy(first, last, this->begin());
     }
   }
-#ifdef __cpp_lib_containers_ranges
+#else
+  template <class Itr> constexpr inplace_vector(Itr first, Itr last) : base() {
```

Comment:
Do you think it's worth constraining `Itr` here using old school SFINAE?

Reply:
That's a good question... 
I think we should just leave to SFINAE first....
Pre 17 template constraints are a pain...

Reply:
Egh! I do hate writing enable_if and template magics.

I think we should leave this as is, we can come back and write constraints as cherry-on-top.

There's other things (tests and constexpr) I want to focus on.